docs/insights/api.md

```diff
@@ -0,0 +1,511 @@
+
+## Get a list of Insights reports
+GET: api/{spaceId}/insights/reports
+Returns a paginated list of the Insights reports in the supplied Octopus Deploy space.
+
+### Parameters
+| Name | Description |
+| ---- | ----------- |
+| spaceId | Id of the space |
+| skip | Number of items to skip. Defaults to zero |
+| take | Number of items to take (i.e. the page size). Defaults to 30 |
+
+### Sample response
+```json
+{
+  "ItemType": "InsightsReport",
+  "TotalResults": 2,
+  "ItemsPerPage": 30,
+  "NumberOfPages": 1,
+  "LastPageNumber": 0,
+  "Items": [
+    {
+      "Id": "InsightsReports-1",
+      "SpaceId": "Spaces-1",
+      "Name": "High Volume Report",
+      "Description": "High volume tenanted deployments report",
+      "ProjectGroupIds": ["ProjectGroups-1", "ProjectGroups-2"],
+      "ProjectIds": ["Projects-1", "Projects-2"],
+      "TenantIds": ["Tenants-1", "Tenants-2", "Tenants-3"],
+      "TenantTags": [
+        "Deployment Group/Group A",
+        "Deployment Group/Group B",
+        "Deployment Group/Group C",
+        "Location/Australia",
+        "Location/Asia",
+        "Location/Europe",
+        "Location/US"
+      ],
+      "TenantMode": "TenantedAndUntenanted",
+      "EnvironmentGroups": [
+        {
+          "Name": "Test",
+          "Environments": ["Environments-1", "Environments-2"]
+        },
+        {
+          "Name": "Internal",
+          "Environments": ["Environments-3", "Environments-4"]
+        },
+        {
+          "Name": "Production",
+          "Environments": ["Environments-5", "Environments-6"]
+        }
+      ],
+      "TimeZone": "UTC",
+      "IconId": "map-signs",
+      "IconColor": "#3CA4F3",
+      "Links": {
+        "Self": "/api/Spaces-1/insights/reports/InsightsReports-1",
+        "Logo": "/api/Spaces-1/insights/reports/InsightsReports-1/logo"
+      }
+    }
+  ],
+  "Links": {
+    "Self": "/api/Spaces-1/insights/reports?skip=0&take=30",
+    "Template": "/api/Spaces-1/insights/reports{?skip,take}",
+    "Page.All": "/api/Spaces-1/insights/reports?skip=0&take=2147483647",
+    "Page.Current": "/api/Spaces-1/insights/reports?skip=0&take=30",
+    "Page.Last": "/api/Spaces-1/insights/reports?skip=0&take=30"
+  }
+}
+```
+
+### Properties
+| Property | Description |
+| -------- | ----------- |
+| ItemType | The type of resource - for Insights Reports, this will always be  "InsightsReport” |
+| TotalResults | The total number of reports |
+| ItemsPerPage | The number of reports per page |
+| NumberOfPages | The total number of pages |
+| LastPageNumber | The number of the last page of reports, counting from zero |
+| Items | An array of the reports returned for the current page.  See the section [Get a specific Insights Report](#get-a-specific-insights-report) for details of the report structure |
+| Links | Links to other resources.  See REST API Links https://octopus.com/docs/octopus-rest-api#api-links |
+
+
+## Create an Insights report
+POST: api/{spaceId}/insights/reports
```

Comment:
I don't know if the leading slash should be added. I checked my API, and it included one

```suggestion
POST: `/api/{spaceId}/insights/reports`
```